api/terms/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en" prefix="skos: http://www.w3.org/2004/02/skos/core#">
+  <head>
+    <meta charset="UTF-8">
+    <title>iRail terminology</title>
+  </head>
+  <body typeof="skos:ConceptScheme" resource="https://api.irail.be/terms/">
```

Comment:
I'd also prefer the URI without a trailing slash. Just personal preference in the case of iRail, nothing written in stone...